src/dbsync/api.rs

```diff
@@ -38,6 +38,28 @@ pub fn get_utxo_tokens(
     Ok(multi_assets)
 }
 
+// stxo = spent transaction output (as opposed to utxo, which is unspent)
+pub fn get_stxo_tokens(
```

Comment:
That is not a good name, we discussed in chat STxO is just a name we defined for the two of us to help you distinguish the concepts. Please name it according to its purpose as this will make no sense for any Cardano developer.
For example:
"get_txo_tokens"
"get_multiassets_for_historic_utxos"
...